src/main/java/by/dragonsurvivalteam/dragonsurvival/client/emotes/EmoteHandler.java

```diff
@@ -121,7 +121,7 @@ public static void playerTick(PlayerTickEvent event){
 
 	@SubscribeEvent
 	public static void playerAttacked(LivingHurtEvent event){
-		EmoteMenuHandler.clearEmotes();
+		EmoteMenuHandler.clearEmotes(event.getEntity());
```

Comment:
would recommend checking for player here since there is no need to handle any other entities
```java
if (event.getEntity() instanceof Player player) {
    EmoteMenuHandler.clearEmotes(player);
}
```